### PR TITLE
FiftyOne Teams 1.7/FiftyOne v0.24.0 release notes

### PR DESCRIPTION
**Pull request by brimoor**
Screenshot 2024-05-28 at 10 33 35 AM

## Summary by CodeRabbit

- **New Features**
  - Added support for 3D meshes and geometries.
  - Introduced new models in the model zoo, including Ultralytics YOLOv8 and Grounding DINO.
  - Enabled saving custom workspaces.
  - Added a media fallback option in the dataset App config.
  - Introduced a new anomaly detection tutorial.
  - Added support for non-sequential category IDs in COCO format.

- **Improvements**
  - Enhanced path normalization for dataset imports.
  - Improved handling of None-valued tags fields.
  - Allowed sidebar changes during loading states.
  - Updated utilities for creating patches, frames, and clips datasets.

- **Bug Fixes**
  - Fixed overlay processing for empty label lists.
  - Resolved concurrency issues when updating/deleting runs and views.
  - Fixed a timezone bug with DateField for GMT+ users.
  - Corrected a bug with YOLO-NAS inference.
  - Addressed a regression related to dynamically grouped views.

- **Documentation**
  - Added images and updated information in the user guide and roles and permissions documentation.
  - Included details about customizing tooltip attributes in the label editor.

### CODE REVIEW

**Review by ehofesmann — APPROVED** [review]
It looks good to me! I added two additional bullets for features that I think are relevant to call out, but up to you if you want to include them or left them out for a reason.

File: docs/source/release-notes.rst [hunk 1]
```diff
@@ -3,6 +3,118 @@ FiftyOne Release Notes
...
+- :meth:`launch_app() <fiftyone.core.session.launch_app>` now respects the
+  current  :meth:`group_slice <fiftyone.core.dataset.Dataset.group_slice>`
+  when loading grouped datasets
+  `#4423 <https://github.com/voxel51/fiftyone/pull/4423>`_
+- Allow sidebar changes during
+  :ref:`lightning loading states <app-lightning-mode>`
+  `#4319 <https://github.com/voxel51/fiftyone/pull/4319>`_
+- Fixed overlay processing for empty label lists
+  `#4345 <https://github.com/voxel51/fiftyone/pull/4345>`_
+- Fixed ``support`` filtering in the sample model for |TemporalDetections|
```
Comment (ehofesmann) [inline docs/source/release-notes.rst:58]:
```suggestion
- Fixed ``support`` filtering in the sample modal for |TemporalDetections|
```

File: docs/source/release-notes.rst [hunk 2]
```diff
@@ -3,6 +3,118 @@ FiftyOne Release Notes
...
+  and :func:`make_clips_dataset() <fiftyone.core.clips.make_clips_dataset>`
+  utilities can now be directly called
+  `#4416 <https://github.com/voxel51/fiftyone/pull/4416>`_
+
+Plugins
+
+- Added a
+  :meth:`set_spaces() <fiftyone.operators.operations.Operations.set_spaces>`
+  method for setting the current spaces layout from operators
+  `#4381 <https://github.com/voxel51/fiftyone/pull/4381>`_
```
[the remark this answers is not in the document]
Comment (brimoor) [reply docs/source/release-notes.rst:112]: Missed this one, thanks!

File: docs/source/release-notes.rst (hunk 1 above)
Comment (brimoor) [reply docs/source/release-notes.rst]: Fixed, thanks 🙇

**Pull request merged**